docker/vector.yaml

```diff
@@ -0,0 +1,201 @@
+api:
```

Comment:
this should probably be added to the `volumes/vector` folder - similar to what we've done with Kong

Reply:
Will change 👍